src/lxc/commands.c

```diff
@@ -167,7 +169,7 @@ static int lxc_cmd_rsp_recv(int sock, struct lxc_cmd_rr *cmd)
 		errno = EFBIG;
 		SYSERROR("Response data for command \"%s\" is too long: %d bytes > %d",
```

Comment:
Specifically this is an example where the explicit `EFBIG` doesn't really provide any additional info.